src/handlers/payout/action.ts

```diff
@@ -90,7 +90,10 @@ export const handleIssueClosed = async () => {
     logger.info(`Penalty removed`);
     return;
   }
-
+  if (privateKey == "") {
```

Comment:
I guess this is acceptable because in our config loader, the default will be an empty string although I think it should be set to `null`

Reply:
should i make the changes? And will it cancel auto merge?

Reply:
I made a new issue.